utils/validate-install-plan.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { fetchPaginatedGHResults } = require('./github-api-helpers');
+const { findMainInstallConfigFiles, readQuickstartFile } = require('./helpers');
+const path = require('path');
+
+const url = process.argv[2];
+
+const CONFIG_REGEXP = new RegExp('quickstarts/.+/config.+(yml|yaml|json)');
+
+const getAllInstallPlanIds = () => {
```

Comment:
Could we add comment blocks with description, parameters and return types  for the functions in this file to match the repo's convention?

Thinking of these: https://github.com/newrelic/newrelic-quickstarts/blob/main/utils/check-missing-logos.js#L6-L10